safe-room.lic

```diff
@@ -26,9 +26,24 @@ class SafeRoom
 
     return unless args.force || need_healing?
 
-    if use_pc_empath?(settings.safe_room_id, settings.safe_room_empath)
-      ensure_copper_on_hand(settings.safe_room_tip_threshold || 0, settings)
-      tip(settings.safe_room_tip_threshold, settings.safe_room_tip_amount, settings.safe_room_empath)
+    pc_empaths = settings.safe_room_empaths || []
+    if settings.safe_room_id && settings.safe_room_empath
+      # Add to pc_empaths list if its not in there already
+      pc_empaths << { 'name' => settings.safe_room_empath, 'id' => settings.safe_room_id } unless pc_empaths.any? { |empath| empath['name'] == settings.safe_room_empath && empath['id'] == settings.safe_room_id }
+    end
+
+    if !pc_empaths.empty?
+      # Sort pc empaths list by nearest first
+      sort_destinations(pc_empaths.map { |empath| empath['id'] })
+        .map { |room| pc_empaths.select { |empath| empath['id'] == room } }
+        .flatten
+        .each do |empath|
+        next unless DRRoom.pcs.include?(empath['name'].capitalize!) || in_game?(empath['name'])
+        next unless use_pc_empath?(empath['id'], empath['name'])
+        ensure_copper_on_hand(settings.safe_room_tip_threshold || 0, settings)
```

Comment:
The `|| 0` looks strange, maybe that should be part of `ensure_copper_on_hand`?

Reply:
What about changing the function prototype to have `safe_room_tip_threshold=0`?